app/assets/javascripts/visualizations/highvis/bar.coffee

```diff
@@ -75,6 +75,8 @@ $ ->
              }]
           xAxis:
             type: 'category'
+          legend:
+            enabled: if $(window).width() > 700 && data.groups.length < 20 then true else false
```

Comment:
Use `and` not `&&`
Also the then true else false implies this would be the same line:
``enabled: $(window).width() > 700 and data.groups.length < 20``